ui/app/AppLayouts/Onboarding2/KeycardCreateProfileFlow.qml

```diff
@@ -113,17 +113,13 @@ SQUtils.QObject {
     Component {
         id: backupSeedRevealPage
         BackupSeedphraseReveal {
-            Component.onCompleted: {
-                try {
-                    d.mnemonic = root.generateMnemonic()
-                    root.seedphraseSubmitted(d.mnemonic)
-                } catch (e) {
-                    console.error('failed to generate mnemonic', e)
-                }
-            }
+            Component.onCompleted: d.mnemonic = root.generateMnemonic()
```

Comment:
`generateMnemonic` is fetching the mnemonic from the device. It's not fully correct to assume it always works. And shouldn't be used in blocking fashion as well imo.

Reply:
Probably correct, but that's something that we had here even before

Reply:
I just removed the `try/except` block as that was there for the `JSON.parse` which got removed in the meantime

Reply:
![image](https://github.com/user-attachments/assets/4dc0f798-3212-4d9c-bf24-cc4f1e25d1c0)

Mnemonic generation is called here when continue is clicked. The communication may always fail. In extreme cases the card may be removed before clicking.

Reply:
Ye ye, I get it; that's not what this PR is about though; worth filing a separate issue.

Reply:
https://github.com/status-im/status-desktop/issues/17299